texts/rode_08.xml

```diff
@@ -440,7 +440,7 @@ sat frem paa Forfærdelsens Alter.</p>
             <p xml:id="idm139893992991840">O Tempel for Gaadernes Gaade,<lb xml:id="idm139893992991536"/>
 som presser os Blodet mod Bringen,<lb xml:id="idm139893992991232"/>
 mens Hjertet maa bede om Naade<lb xml:id="idm139893992990928"/>
-og omfavner <hi rend="italics" xml:id="idm139893992990640">Intet og Ingen</hi>.</p>
+og omfavner <hi rend="italics" xml:id="idm139893992990640">Intet</i> og <hi rend="italics">Ingen</hi>.</p>
```

Comment:
Här fick vi in ett </i> som avslut till <hi>, sorry doesn't work. Fixat i här!

Reply:
Hovsa! Beklager.

Reply:
Nema problema! :+1: